LoopKitUI/ViewModels/DisplayGlucosePreference.swift

```diff
@@ -23,7 +23,6 @@ public class DisplayGlucosePreference: ObservableObject {
         self.unit = displayGlucoseUnit
         self.rateUnit = rateUnit
         let formatter = QuantityFormatter(for: displayGlucoseUnit)
-        formatter.unitStyle = .short
```

Comment:
curious if you tested with `mmol/L`. I think this code was added to address an issue there.